system/greenhouse-ccloud/templates/_helper.tpl

```diff
@@ -27,6 +27,14 @@
 {{- end -}}
 {{- end -}}
 {{- end -}}
+{{- range $addStorePrefix := $cluster.additionalStores }} 
+{{- range $stores }}
+{{- $endpointSplit := split "." . }}
+{{- $region := $endpointSplit._2 }}
```

Comment:
if the `additionalStore` is `thanos-openstack`, how is this splitable by `.`?

Reply:
We range over 1. to add 2.

1. We collect all stores along ingressPlugins. e.g. `thanos-grpc.$ingressPluginRecordName:443`
2. If `additionalStore`, we add e.g. thanos-openstack for the same region. Result will be e.g. `thanos-openstack-grpc.$region.$baseDomainPort`